src/main/java/seedu/recipe/model/recipe/ingredient/IngredientQuantifier.java

```diff
@@ -0,0 +1,52 @@
+package seedu.recipe.model.recipe.ingredient;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+
+/**
+ * Represents a container for the values stored with respect to an Ingredient,
+ * within a Recipe instance.
+ */
+public class IngredientQuantifier {
+    private final Optional<IngredientQuantity> quantityContainer;
+    private final Optional<String> commonNameContainer;
+    private final List<String> estimatedQuantity = new ArrayList<>();
```

Comment:
i think it suffices to have `estimatedQuantity` as an `Optional<String>` instead of a `List<String>` actually because keeping it as a list of estimated quantities doesn't really add value as we're not implementing any features related to ingredient quantities